config/project.cmake

```diff
@@ -38,11 +38,12 @@ else()
   message(STATUS "Note: using 32 bit integer ids.")
 endif()
 
+cinch_load_extras()
 #------------------------------------------------------------------------------#
 # Add library targets
 #------------------------------------------------------------------------------#
 
-cinch_add_library_target(flecsi-sp specializations)
+cinch_add_library_target(flecsi-sp flecsi-sp)
```

Comment:
I think I would keep `cinch_add_library_target(flecsi-sp specializations)`, and just replace `#include "flecsi-sp/...."` with `#include "specialiations/...."`